readyplayerme/rpm-unity-sdk-core
# feat: ask, if user wants to continue editing their previous avatar, when they stopped at the middle

![image](https://github.com/readyplayerme/rpm-unity-sdk-core/assets/107070960/44090151-074a-4595-b4eb-45e1ca08e2c6)

## [SDK-723](https://ready-player-me.atlassian.net/browse/SDK-723)

## Description

-   Ask, if the user wants to continue editing their avatar if they closed the avatar creator while editing and open it again

## How to Test

- log in
- Start modifying/creating new avatar
- Close scene (Exit play mode)
- Run again and see, that the pop-up that asks to continue editing is shown






[SDK-723]: https://ready-player-me.atlassian.net/browse/SDK-723?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
